spec/rubocop/cop/rspec/example_wording_spec.rb

```diff
@@ -187,4 +187,29 @@
       end
     RUBY
   end
+
+  it 'flags an unclear description' do
+    expect_offense(<<-'RUBY')
+      it "throws error" \
+          ^^^^^^^^^^^^^^^ Your test should be more descriptive.
+        "with #{object}" do
```

Comment:
Those should be examples of what the cop should ignore.
The full message is "throws error with a message". Why is this an offence?

It seems that:
 - we don't consider articles e.g. "throws an error"
 - we don't inspect complete messages, but just the first part of it

Reply:
Thanks for bringing up the article part! you are right that we don't consider articles! I went down a rabbit hole of trying to ignore the articles and strip the phrase to just ["throws error"] from ["throws error, throws an error"] etc... But then I realized there could be grammatical errors that get missed from it. so I'm a little confused at what to do. I removed the phrase for the time being though. 

as for #2, I thought include? checks for whole phrases. so we will ignore "throws an error with a message" but the cop will sound an alarm for "throws error". Is there something I'm missing?